todo_app/data/roleChecker.py

```diff
@@ -0,0 +1,11 @@
+from flask_login import current_user
+from flask import redirect
+import os
+
+def ReadWriteNeeded(func):
+    def wrapper(*args, **kwargs):
+        if current_user.role == 'Read/Write' or os.getenv('LOGIN_DISABLED') == 'True':
+            return func(*args, **kwargs)
+        else: return redirect('/')
+    wrapper.__name__ = func.__name__
```

Comment:
Nice find here - an alternative is to use the `functools.wraps` decorator which achieves a similar thing without having to expose the magic quite as much:
```
from functools import wraps

def ReadWriteNeeded(func):
    @wraps(func)
    def wrapper(*args, **kwargs):
        if current_user.role == 'Read/Write' or os.getenv('LOGIN_DISABLED') == 'True':
            return func(*args, **kwargs)
        else: return redirect('/')
    return wrapper
```